javalin/src/main/java/io/javalin/json/JavalinJackson.kt

```diff
@@ -48,6 +50,16 @@ class JavalinJackson(private var objectMapper: ObjectMapper? = null) : JsonMappe
         }
     }
 
+    override fun handleOutputStream(outputStream: OutputStream, obj: Any, type: Type): Boolean = when (obj) {
+        is Stream<*> -> true.also {
+            mapper.writer().writeValues(outputStream).use { sequenceWriter ->
```

Comment:
Writing to `outputStream` instead of `req().outputStream`, which will get the compressed output stream, whereas the latter is the raw output stream.